### ISSUE TITLE
Community nvALT Usage

### ISSUE THREAD

**Issue by bawigga** [bawigga/nvalt-prime#14]
How do you use nvALT day to day? As a diary? Research Wiki? README documentation? I'd love to hear more about how and why the community uses nvALT.

**Comment by roylockhart** [comment]
I use nvALT :    
+ As a diary
    - What should happen today
    - ToDo (CHECKBOX!)
    - Not schedulded events
    - Comments about the day
    - Research stuff ("oh hay, I added this to the wiki/dissertation and found x)
+Wiki for research. (Added clickable wiki links in the preview window. It's a simple script but it makes my life WAY easier.[Source](http://brettterpstra.com/2013/08/25/clickable-wiki-links-in-the-nvalt-preview/)). Also made an index to stay organized (you know, when you have YEARS of diary entries)
+ To note everything I have in mind : links, blogpost ideas, etc...
+ To read later websites without messing with firefox tabs
+ To store passwords (I have an encrypted database)
+ Recipes, manifestos, tweet drafts...

**Comment by bawigga** [comment]
> Also made an index to stay organized (you know, when you have YEARS of diary entries)

@roylockhart So you have a diary index note that you have to manually add wiki link entries to?

> Added clickable wiki links in the preview window. It's a simple script but it makes my life WAY easier

Would love to add this to nvalt-prime. Want to PR?

**Comment by roylockhart** [comment]
@bawigga Yes, I have a General Index for wiki links. Since I have a lot of diary notes, I created a separated index, which is in the General Index. Wikiception.

Sounds like a lot of work, but I don't need automation for that. I have templates :)

> Would love to add this to nvalt-prime. Want to PR?

Yes of course :)

**Comment by umeyuki** [comment]
I use nvALT: 
-    draft,
-    research wiki
-    GTD(not yet) 

I'm looking forward a note application fit me for a long time.

## Required 

- simply
- markdown  support
- checkbox  
- bring to front hotkey
- plain text 
- tag
- save as dropbox
- can use iPhone app (I use [Write](http://writeapp.net) now)

nvalt-prime has every requirement.

You make my writing life!

**Comment by dg2** [comment]
Hey @bawigga, thanks for your great work on nvalt-prime!! I keep tons of notes with both code and math on them, so I have extended nvalt-prime to support MathJax. It might not be everybody's cup of tea, as it brings in the 168M of the MathJax repo as a submodule so that's why I'm not sending a PR. Happy to contribute it if people find it useful.

**Comment by dmorel** [comment]
@bwiggs thanks for this useful addition. As a programmer, I use it as a simple way to take notes and have them available anywhere. I nearly exclusively use markdown syntax for everything I write now, and nvalt's very simple interface, lightness and great usability suits me just fine. Maybe some syntax coloring in the editing pane would be nice, but the great preview solves 90% of the (non-?)issue.
